packages/components/src/KaizenProvider/docs/KaizenProvider.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { Meta } from '@storybook/addon-docs'
+import { CATEGORIES } from "../../../../../storybook/constants"
+
+<Meta title={`${CATEGORIES.components}/KaizenProvider`} />
```

Comment:
I honestly don't like the `components` category. It's too broad and not useful. Categories like Graphics/Form/Layout are better (separate discussion from this PR though). Not that I know what to group this under though 😂

Reply:
I'd group it under `Global` -- But yeah a wider discussion on this, I know the designers have already come up with groupings